src/grammar-registry.coffee

```diff
@@ -54,9 +53,9 @@ class GrammarRegistry extends FirstMate.GrammarRegistry
 
   getGrammarPathScore: (grammar, filePath) ->
     return -1 unless filePath
-    filePath = filePath.replace(/\\/g, '/') if process.platform is 'win32'
 
-    pathComponents = filePath.toLowerCase().split(PathSplitRegex)
+    pathComponents = filePath.toLowerCase().split(path.sep)
```

Comment:
After sleeping on this I'm no longer sure that this is a clean solution.  `file:///` handlers use forward-slashes, and so might other remote files.  I'm not 100% sure every file path passed to `getGrammarPathScore` already uses that OS's path separator.

/cc @maxbrunsfeld